microSALT/utils/referencer.py

```diff
@@ -135,63 +135,55 @@ def index_db(self, full_dir, suffix):
             self.logger.info("Re-indexed contents of {}".format(full_dir))
 
     def fetch_external(self, force=False):
-        """ Updates reference for data that IS ONLY LINKED to pubMLST """
-        prefix = "https://pubmlst.org"
-        query = urllib.request.urlopen("{}/data/".format(prefix))
-        soup = BeautifulSoup(query, "html.parser")
-        tr_sub = soup.find_all("tr", class_="td1")
-        tr_sub = tr_sub + soup.find_all("tr", class_="td2")
-
-        # Only search every other instance
-        iterator = iter(tr_sub)
-        unfound = True
+        url = "https://pubmlst.org/static/data/dbases.xml"
         try:
-            while unfound:
-                entry = iterator.__next__()
-                # Gather general info from first object
-                sample = entry.get_text().split("\n")
-                organ = sample[1].lower().replace(" ", "_")
-                # In order to get ecoli #1
+            query = urllib.request.urlopen(url).read()
+            root = ET.fromstring(query)
+            for entry in root:
+                # Check organism
+                species = entry.text
+                organ = species.strip().lower().replace(" ", "_") 
                 if "escherichia_coli" in organ and "#1" in organ:
                     organ = organ[:-2]
+                if organ not in self.organisms:
```

Comment:
Placement of not like this makes it invalid. Effectively it now says "if organ in self.organisms"